source/developers-guide/shopware-5-search-bundle/index.md

```diff
@@ -140,20 +140,44 @@ Each `ConditionHandler` must implement the `Shopware\Bundle\SearchBundleDBAL\Con
 The `Shopware_SearchBundleDBAL_Collect_Condition_Handlers` event can be used to register a condition handler class:
 
 ```php
+<?php
 //register condition handlers for dbal search implementation
-Shopware()->Events()->addListener(
-   'Shopware_SearchBundleDBAL_Collect_Condition_Handlers',
-   function() {
-      return new CategoryConditionHandler();
-   }
-);
+
+namespace YourPlugin;
+
+class YourPlugin extends \Shopware\Components\Plugin
+{
+    public static function getSubscribedEvents()
+    {
+        return [
+            'Shopware_SearchBundleDBAL_Collect_Condition_Handlers' => 'onCollectConditionHandlers'
```

Comment:
please do not describe the collect event, but the `tag` in the services.xml instead